### PR TITLE
fix: Fixed menu items exceeded canvas issue.

### LINKED ISSUE #11684: [Bug]: More menu items exceed the canvas
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

 In menu button widget when new Menu items are added in property pane the bottom border is not visible after the 5th new menu item
 
When more number of menu items are added the items move out of canvas 
image

https://loom.com/share/3398a3caca3a4b0d95cc4b07d0f846c3

### Steps To Reproduce

1. Drag and drop Menu button widget
2. In property pane add new buttons

### Environment

Production

### Version

Cloud

### PR DESCRIPTION
**Pull request by MuhammadAakash**
## Description
Fixed the issue of when we added many menu items. So, they exceed the canvas. So, i fixed them by adding scroll bar on it and added the height for it so, it will not exceed the canvas.

Fixes #11684 

## Media
https://www.loom.com/share/5a1f9b4b2f294ad8a22b9de4b89f112e

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?
Create a menu widget and add at least 20 to 30 items to it. So, previously it exceed the canvas length but now it's working perfectly fine.  

### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: app/client/src/widgets/MenuButtonWidget/component/index.tsx [hunk 1]
```diff
@@ -53,7 +53,8 @@ const PopoverStyles = createGlobalStyle<{
     margin-bottom: 8px !important;
     border-radius: ${({ borderRadius }) =>
       borderRadius >= THEMEING_TEXT_SIZES.lg ? `0.375rem` : borderRadius};
-    overflow: hidden;
+    overflow-y: scroll;
+    height: 384px;
```
Comment (somangshu) [inline app/client/src/widgets/MenuButtonWidget/component/index.tsx:57]: I think it should rather be max-height. Checkout the behaviour on the popup, it has empty space for lesser number of items. cc @jsartisan what do you say?

Screenshot 2023-02-14 at 11 53 18 AM

**Review by somangshu — CHANGES_REQUESTED** [review]

File: app/client/src/widgets/MenuButtonWidget/component/index.tsx (hunk 1 above)
Comment (MuhammadAakash) [reply app/client/src/widgets/MenuButtonWidget/component/index.tsx:57]: exactly, you are right.
I will add the commit with the necessary changes.

**Review by somangshu — APPROVED** [review]
LGMT, the change has fixed the issue

@jsartisan kindly review and verify

**Review by jsartisan — APPROVED** [review]

**Comment by jsartisan** [comment]
@MuhammadAakash can you take pull of latest release on your branch?

**Comment by MuhammadAakash** [comment]
@jsartisan 
Sure, I will do this tonight.

**Comment by MuhammadAakash** [comment]
@jsartisan 
After pulling the release branch into my current branch, I got that no files were changed.

**Comment by jsartisan** [comment]
@MuhammadAakash You are taking a pull of release from your forked repo. You have to take a pull from appsmith's release branch. Basically your repo's release don't have the latest release code. A sync is required.

**Comment by MuhammadAakash** [comment]
@jsartisan 
Sorry for being late. I have merged the release branch with my current branch. Please check

**Comment by MuhammadAakash** [comment]
@somangshu 
Are there any other changes which need to be completed?

**Pull request merged**